flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is very disappointing! why not building from source to also suppose other platforms ?

Reply:
The build process is kinda convoluted: it require the whole `nix` setup to build two haskell libraries (which can take up to 12 hours, because _everything_ is building from source), the `Compose Multiplatform` setup (which I'm not the expert of) with java, kotlin and all of that stuff.

Not to mention we're currently doesn't support `aarch64` on desktop :)